### DISCUSSION TITLE
nrm problem

### DISCUSSION THREAD

**Discussion by xgz123** [verdaccio/verdaccio#2257] [Q&A] [answered]
`nrm add local mydomain.com:4873`
output:
> Registry has been set to: https://registry.verdaccio.org/

then `nrm use npm`
it still prints
> Registry has been set to: https://registry.verdaccio.org/

why?

**Comment by juanpicado** [comment, marked as answer]
maybe if you explain why you have set `https://registry.verdaccio.org/` as registry. btw If you have cloned this project, the `.npmrc` points to `https://registry.verdaccio.org/`
